src/backends/sentry_backend_breakpad.cpp

```diff
@@ -118,13 +149,18 @@ sentry__breakpad_backend_callback(
     sentry_capture_event(event);
 
     // after capturing the crash event, try to dump all the in-flight data of
-    // the previous transport
-    if (transport) {
-        sentry__transport_dump_queue(transport);
+    // the previous transports
+    if (transport_disk) {
+        sentry__transport_dump_queue(transport_disk);
```

Comment:
While this is not *wrong*, you don’t need it since the disk transport is completely sync.

Reply:
Yeah, I put it here just in case the logic of callback sometime gets more complicated.